packages/access-api/test/helpers/ucanto-test-utils.js

```diff
@@ -92,6 +135,42 @@ export async function spaceRegistrationInvocation(
   return redeem
 }
 
+/**
+ * get an access-api invocation that will register an account.
+ * This is useful e.g. because some functionality (e.g. access/delegate)
+ * will fail unless the space is registered.
+ *
+ * @param {Ucanto.Signer<Ucanto.DID>} service - issues voucher/redeem. e.g. could be the same signer as access-api env.PRIVATE_KEY
+ * @param {Ucanto.DID<'mailto'>} account
+ * @param {Ucanto.DID<'key'>} agent
+ * @param {Ucanto.Principal} audience - audience of the invocation. often is same as issuer
+ * @param {number} lifetimeInSeconds
+ */
+export async function accountRegistrationInvocation(
+  service,
+  account,
+  agent,
+  audience = service,
+  lifetimeInSeconds = 60 * 15
+) {
+  const registerSpace = await Access.confirm
```

Comment:
should this be named `registerAccount` ?

Reply:
ty https://github.com/web3-storage/w3protocol/pull/530/commits/b4fc3c7c9e9c38564487b24d896b3643b06fc5b6